convert2rhel/pkghandler.py

```diff
@@ -316,7 +317,7 @@ def print_pkg_info(pkgs):
             from_repo = pkg._from_repo
 
         pkg_list += "%-*s  %-*s  %s" % (max_nvra_length, get_pkg_nvra(pkg),
-                                        max_packager_length, get_packager(pkg), from_repo) + "\n"
+                                        max_packager_length, get_vendor(pkg), from_repo) + "\n"
```

Comment:
Need to check for vendor and have fallback here. <8.3 will otherwise only show N/A. I think it would be better if we have another function that does this check for us and calls `get_packager` and `get_vendor` as needed